packages/dialect-react-sdk/hooks/useNotificationChannelDappSubscription.ts

```diff
@@ -26,11 +25,11 @@ interface UseNotificationChannelDappSubscriptionValue {
 }
 
 const useNotificationChannelDappSubscription = ({
-  dappPublicKey: dappPublicKeyOverride,
+  dappAddress: dappAddressOverride,
   addressType,
 }: UseNotificationChannelDappSubscriptionParams): UseNotificationChannelDappSubscriptionValue => {
-  const { dappAddress: globalDappPublicKey } = useDialectDapp();
-  const dappPublicKey = dappPublicKeyOverride || globalDappPublicKey;
+  const { dappAddress: globalDappAddress } = useDialectDapp();
```

Comment:
I think for backward compatibility useDialectDapp(true) is needed here: previous version did not throw exception if dappAddress was not defined

Reply:
this is an extremely good catch! thanks